infra/docker-compose/.env.sample

```diff
@@ -0,0 +1,16 @@
+COMPOSE_PROJECT_NAME=feast
+
+FEAST_VERSION=0.3.0
+FEAST_GCP_SERVICE_ACCOUNT_KEY_PATH=/etc/gcloud/service-accounts/key.json
+
+FEAST_CORE_IMAGE=gcr.io/kf-feast/feast-core
+FEAST_CORE_CONFIG=override
+FEAST_CORE_CONFIG_PATH=/etc/feast/feast-core/override.yaml
+FEAST_CORE_GCP_SERVICE_ACCOUNT_KEY=placeholder
+
+FEAST_SERVING_IMAGE=gcr.io/kf-feast/feast-serving
+FEAST_SERVING_STORE_CONFIG=redis-store
+FEAST_SERVING_STORE_CONFIG_PATH=/etc/feast/feast-serving/store.yaml
+FEAST_SERVING_GCP_SERVICE_ACCOUNT_KEY=placeholder
+
+FEAST_JUPYTER_IMAGE=gcr.io/kf-feast/feast-jupyter
```

Comment:
Not finding this. Did you end up pushing it, or should I help with that?

Reply:
I haven't pushed it yet. The current dockerfile always build from master branch, which is fine, but for production we should probably install via pip instead. For which versions should the Feast jupyter images be available? All versions after and including 0.3? (0.3.0, 0.3.2)

Reply:
Yes, ideally all versions.